src/CommunityToolkit.Maui/Views/DrawingView/DrawingView.shared.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System.Collections.ObjectModel;
+using System.Windows.Input;
+using CommunityToolkit.Maui.Core;
+using CommunityToolkit.Maui.Core.Views;
+
+namespace CommunityToolkit.Maui.Views;
+
+/// <summary>
+/// The DrawingView allows you to draw one or multiple lines on a canvas
+/// </summary>
+public class DrawingView : View, IDrawingView
```

Comment:
Let's add `UnsupportedOSPlatformAttribute` like @pictos suggested

```suggestion
﻿using System.Collections.ObjectModel;
﻿using System.Runtime.Versioning;
using System.Windows.Input;
using CommunityToolkit.Maui.Core;
using CommunityToolkit.Maui.Core.Views;

namespace CommunityToolkit.Maui.Views;

/// <summary>
/// The DrawingView allows you to draw one or multiple lines on a canvas
/// </summary>
[UnsupportedOSPlatform("windows")]
public class DrawingView : View, IDrawingView
```